modules/ossm-configuring-jaeger.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+// * service_mesh/service_mesh_install/customizing-installation-ossm.adoc
+
+[id="ossm-configuring-jaeger_{context}"]
+= Configuring Jaeger
+
+When the {ProductShortName} Operator creates the `ServiceMeshControlPlane` resource it also creates the Jaeger resource. The Jaeger Operator then uses this object when creating Jaeger instances.
+
+The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as follows:
+
+.Default Jaeger parameters
+[source,yaml]
+----
+  apiVersion: maistra.io/v1
+  kind: ServiceMeshControlPlane
+  spec:
+    istio:
+      tracing:
+        enabled: true
+        ingress:
+          enabled: true
+
+----
+
+.Jaeger parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+|tracing
+   enabled
+|This enables or disables tracing in {ProductShortName}. Jaeger is installed by default. If you do not want to install Jaeger, change the `enabled` value to `false`.
+|`true`/`false`
+|`true`
+
+|ingress
+   enabled
+|This enables/disables ingress.
+|`true`/`false`
+|`true`
+|===
+
+[id="ossm-configuring-jaeger-elasticsearch_{context}"]
+== Configuring Elasticsearch
+
+Jaeger can be configured for different storage backends:
+
+* *Memory* - Simple in-memory storage, only recommended for development, demo, or testing purposes. This is the default option for the `AllInOne` deployment strategy. Do NOT use for production environments.
+* *Elasticsearch* - For production use. This is the default option for the `Production` deployment strategy.
+
+[NOTE]
+====
+For this release, the only supported storage option for production use is Elasticsearch.
+====
+
+Elasticsearch is a memory intensive application. The initial set of nodes created by the {product-title} installation may not be large enough to support the Elasticsearch cluster. Additional nodes must be added to the  cluster if you want to run with the recommended amount (or more) memory. Each ES node can operate with a lower memory setting though this is not recommended for production deployments.
+
+You should modify the default Elasticsearch configuration to match your use case. You can adjust both the CPU and memory limits for each component by modifying the resources block with valid memory and CPU values.
+
+
+.Default Jaeger parameters
```

Comment:
Default Elasticsearch parameters when using the `production-elasticsearch` jaeger template.